src/controllers/loginController.ts

```diff
@@ -29,7 +30,7 @@ const loginController = async (req: Request, res: Response) => {
         template: 'email',
         context: {
           name,
-          verificationLink: `${process.env.ROOT_URL}/api/users/verify-email/${verificationToken}`,
+          verificationLink: `${process.env.SWAGGER_ROOT_URL}api/users/verify-email/${verificationToken}`,
```

Comment:
please don't touch something that is already working

Reply:
the changed one is one that works fabrice you can ask honore musabe he was the one who told us to do the firatone was crashing